### PR TITLE
[15.0][ADD] partner_readonly_security: New module

### PR DESCRIPTION
**Pull request by victoralmau**
New module

Please @pedrobaeza and  @carolinafernandez-tecnativa can you review it?

@Tecnativa TT48722

### CODE REVIEW

**Comment by carolinafernandez-tecnativa** [comment]
Functional review LGTM,

Only a suggestion, when user has no rights to create partner appears this error message

image

What do you think adding more friendly message for user like "Sorry, you are not allowed to create/edit partners. Please contact your admnistrator for further information" 

Thanks!!

**Comment by HaraldPanten** [comment]
Hi Victor, are you sure that this can't be done by standard?

I'm not really sure if this module is really needed. Could you give us more details of the use case that you have?

THX.

**Comment by pedrobaeza** [comment]
@HaraldPanten we want that a sales manager doesn't edit partners. This can't be done with standard. Maybe the use case can be added to CONTEXT.rst though.

**Comment by victoralmau** [comment]
Changes done.

**Comment by HaraldPanten** [comment]
> @HaraldPanten we want that a sales manager doesn't edit partners. This can't be done with standard. Maybe the use case can be added to CONTEXT.rst though.

Oh, understood! 👍

File: partner_readonly_security/i18n/partner_readonly_security.pot [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#: model:res.groups,name:partner_readonly_security.group_partner_edition
+msgid "Partner edition"
+msgstr ""
+
+#. module: partner_readonly_security
+#: code:addons/partner_readonly_security/models/res_partner.py:0
+#, python-format
+msgid ""
+"Sorry, you are not allowed to create/edit partners. Please contact your "
+"admnistrator for further information."
```
Comment (HaraldPanten) [inline partner_readonly_security/i18n/partner_readonly_security.pot]:
```suggestion
"administrator for further information."
```

**Review by carolinafernandez-tecnativa — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
Without testing deeply, seems to be OK.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**